### DISCUSSION TITLE
Setting the Categories Field.

### DISCUSSION THREAD

**Discussion by louiss0** [estruyf/vscode-front-matter#695] [Ideas]
I wish for a setting where I can set the Categories field. I just discovered that the Categories field is a field that uses array's only.
It does not allow for choices. It only allows the `categories:` field to be used for it. But my app does not do that it's one category per post. I use tags for topics and Categories for structure.

I'm looking for this. 
 
```json
{
          "title": "Category",
          "name": "category",
          "type": "choice",
          "isCategoriesField": true,
          "choices": [
            "Segment",
            "Standalone",
            "Tutorial"
          ]
        },
```

**Comment by estruyf** [comment]
Hi @louiss0, thanks for the suggestion/question. Could the following enhancement to the taxonomy fields fix the issue for you: https://github.com/estruyf/vscode-front-matter/issues/273 ?

**Reply by louiss0** [reply]
That does solve my problem. My solution would have required less boilerplate though

**Reply by estruyf** [reply]
That is true, but it would also mean a significant change in the core logic of the taxonomy fields logic.

What I like about it is that it provides some extra flexibility in the field configuration, so I am definitely open to have it as an enhancement to the tag/category fields in the future.
